text/1003-deprecation-import-ember-from-ember.md

```diff
@@ -0,0 +1,647 @@
+---
+stage: accepted
+start-date: 2024-01-22T00:00:00.000Z
+release-date:
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1003 
+project-link:
+---
+
+<!---
+Directions for above:
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+-->
+
+<-- Replace "RFC title" with the title of your RFC -->
+# Deprecate `import Ember from 'ember'; 
+
+## Summary
+
+This RFC proprosing deprecating all APIs that have module-based replacements, as described in [RFC #176](https://rfcs.emberjs.com/id/0176-javascript-module-api) as well as other `Ember.*` apis that are no longer needed.
+
+## Motivation
+
+The `import Ember from 'ember';` set of APIs is implementn as a barrel file, and properly optimizing barrel files [is a lot of work, requiring integration with build time tools](https://vercel.com/blog/how-we-optimized-package-imports-in-next-js).
+
+By removing this set of exports, we have an opportunity to shrink some apps (as some APIs are not used), improving the load performance of ember apps -- and we removing all of these gives us a chance to have a better grasp of what we can get rid of permananently.
+
+Many of these APIs already have alternatives, and those will be called out explicitly in the _Transition Path_ below.
+
+## Transition Path
```

Comment:
I think after I get through this list, I'm going to convert the data that's just "same thing from imported from elsewhere" into a table, so that way the "reference" is just concise, and mostly re-hashing (verifying?) RFC#176